src/guide/contributing-to-an-open-jdk-project.md

```diff
@@ -8,6 +8,12 @@ If you have a success story where Java solved your problem, or if you successful
 
 If you're in a position to choose what programming language to use in a project, in a tutorial, or in a class, you have the power to enlarge the Java community in a very direct way, and your colleagues or students will get an opportunity to learn one of the most used programming languages in the world.
 
+## Requesting a change
+
+If you want to request a change to the JDK, but don't have the intention to implement/contribute to the change yourself, you should always report it at [bugreport.java.com](bugreport.java.com). This is also the case if you find a change that you would like to see backported to an earlier JDK release. **Never** reach out directly to any developer. Also, please note that the [OpenJDK mailing lists](#mailing-lists) are intended for developers and contributors. These lists are not support channels or channels for requesting changes.
```

Comment:
It's still unclear what a person should do to request backporting a fix. Is it recommended to submit a new bug?